packages/core/package.json

```diff
@@ -18,7 +18,7 @@
     "type": "git"
   },
   "dependencies": {
-    "tslib": "^1.7.1"
+    "tslib": "^2.0.3"
```

Comment:
I would like to avoid releasing a major version so instead supporting both versions would work I think :thinking:, so could you please change it to:

```json
"tslib": "^1.7.1 || ^2.0.3"
```

Reply:
Sure, done

Reply:
Should I also change version from 6.0.0 to something like 5.2.0?

Reply:
Updated